src/main/kotlin/bakalarka/elearningplatform/model/ROLES.kt

```diff
@@ -0,0 +1,7 @@
+package bakalarka.elearningplatform.model
+
+enum class ROLES(val code: String) {
+    USER("rol_K4GNrsvT2qTOc4Js"),
```

Comment:
Tie IDcka by sa mali podla mna nacitavat z `application.properties`. Hmm, ale mozno to nie je take jednoduche. Skus pozriet, ci sa to nejak da.

Reply:
No podarilo sa mi nacitavat veci z application.properties, vid v classe Management, ale imho je riesenie cez enum lepsie, minimalne pokial by to trebalo nacitavat tak ako sa mi to podarilo tam :D .. mozme zhodnotit ked budeme volat

Reply:
Jo, zial v tom Enume neviem ako to uplne vyriesit :/ Mozme skusit rychlo pozriet, ked budeme volat. Mozno toto je riesenie - https://www.baeldung.com/spring-value-annotation alebo https://stackoverflow.com/questions/45192373/how-to-assign-a-value-from-application-properties-to-a-static-variable